### DISCUSSION TITLE
Support for Kibana 8.12

### DISCUSSION THREAD

**Discussion by camusyan** [fbaligand/kibana-enhanced-table#345] [General]
Hello Fabien,
First, I want to thank you for developing and sharing this very useful Kibana plugin.
Recently, I upgraded my ELK platform from 8.2.3 to 8.12.1 but I forgot to check first if your plugin was compatible to this new Kibana version and I discover that the last version supported was the 8.11.
By any chance, do you plan to propose in the coming weeks a new version of your plugin to support 8.12 versions (8.12.0, 8.12.1 and 8.12.2).
Thanks & regards,
Yann

**Comment by fbaligand** [comment]
Hi Yann,

Well, good news, Enhanced Table is already compatible with Kibana 7.12, since February 1st!
You can get it here:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.14.0

**Reply by camusyan** [reply]
I already tried version 1.14.0 but the plugin installation doesn't work.
My request was about Kibana 8.12, not 7.12.

**Reply by fbaligand** [reply]
Well, sorry for my number mistake. I really wanted to say 8.12.
So I invite you to look all assets presents in this link, and you will see that there are assets for Kibana 8.12.0, 8.12.1 and 8.12.2.

**Reply by fbaligand** [reply]
In case you missed them, here are the links:
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_8.12.0.zip
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_8.12.1.zip
https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.14.0/enhanced-table-1.14.0_8.12.2.zip

**Reply by camusyan** [reply]
Hi Fabien,
My bad, I didn't download the proper zip file.
Thanks for sharing the links. It is much better now :)
Thanks again for your help.
Bye

**Reply by fbaligand** [reply]
Nice, enjoy!
